### ISSUE TITLE
Search menu missing appropriate `aria` attributes

### ISSUE THREAD

**Issue by admturner** [washingtonstateuniversity/hrs.wsu.edu#29]
If using a button-menu combination, need to include a haspopup attribute, as well as expanded states.
